src/Settings.ts

```diff
@@ -31,20 +32,24 @@ export let verbose: boolean;
  * @param configFilePath The path to the configuration file
  * @returns The path to the loaded configuration file
  */
-export function loadConfiguration(configFilePaths: string[]): string {
+export function loadConfiguration(configurations: IConfiguration[]): string {
   let settingsConfigJson: ISettingsConfigJson;
   let loadedSettingsFilePath: string;
 
   // Look through the list of possible loadable config files and try loading
   // them in turn until a valid one is found.
-  configFilePaths.some(configFilePath => {
-    settingsConfigJson = helpers.readSettings<ISettingsConfigJson>("Settings", configFilePath);
+  configurations.some(configuration => {
+    settingsConfigJson = helpers.readSettings<ISettingsConfigJson>(
+      "Settings",
+      configuration.settingsFilePath,
+      configuration.secretsFilePath,
+    );
```

Comment:
@danecreekphotography Would you like me to write any tests for this?

Reply:
Nah. I'm not super picky about automated test coverage.